openedx/features/personalized_learner_schedules/show_answer/show_answer_field_override.py

```diff
@@ -53,4 +53,4 @@ def get(self, block, name, default):
     @classmethod
     def enabled_for(cls, course):
         """ Enabled only for Self-Paced courses using Personalized User Schedules. """
-        return course.self_paced and RELATIVE_DATES_FLAG.is_enabled(course.id)
+        return getattr(course, 'self_paced', False) and RELATIVE_DATES_FLAG.is_enabled(course.id)
```

Comment:
I think self_paced *should* be defined for any course object. You could probably just do `course and course.self_paced and...` but this is fine too.